worlds/messenger/options.py

```diff
@@ -133,7 +133,7 @@ class PlannedShopPrices(OptionDict):
 
 
 @dataclass
-class MessengerOptions(PerGameCommonOptions):
+class MessengerOptions(PerGameCommonOptions, DeathLinkMixin):
```

Comment:
```suggestion
class MessengerOptions(DeathLinkMixin, PerGameCommonOptions):
```

Reply:
Did you check python docs, or did you blindly believe me? :D

Reply:
in our specific case here i don't think it really matters; the order matters for the actual inheritance so there isn't really a "standard" for how it should look, but typically mixins are put before the base class from what i'm able to find. couldn't find any official python docs on mixin specifically though

Reply:
I think left overwrites right, so the more specialized it gets, the farther to the left it should be.

The python docs should mention the inheritance behaviour as MRO, if you have too much free time on your hands :D